content/api/events-ingest.apib

```diff
@@ -55,9 +64,9 @@ Post a new-line delimited list of JSON events.
 
     + Headers
 
-        Authorization: 14ac5499cfdd2bb2859e4476d2e5b1d2bad079bf
-        Content-Encoding:gzip
-        Content-Type:application/x-ndjson
+            Authorization: 14ac5499cfdd2bb2859e4476d2e5b1d2bad079bf
```

Comment:
I assume this isn't a real API key right?

Reply:
yeah, it's the same one used all throughout the API docs